src/lexical/LexicalAnalysis.java

```diff
@@ -194,6 +194,8 @@ public Lexeme nextToken() throws IOException {
         if (estado == 12) {
             if (st.contains(lex.token)) {
                 lex.type = st.find(lex.token);
+                
+                System.out.printf("Symbol Table\n Nome: %s\n Token Type: %s\n", lex.token,lex.type);
```

Comment:
Eu ainda to achando que era pra ser o de baixo (`NAME` - ou seja, os identificadores - pra ser colocado na tabela, não os valores) :thinking: